src/Commands/GenerateDocumentation.php

```diff
@@ -60,9 +65,10 @@ public function handle()
         Flags::$shouldBeVerbose = $this->option('verbose');
 
         $this->docConfig = new DocumentationConfig(config('apidoc'));
+        $this->baseUrl = $this->docConfig->get('base_url') ?? config('app.url');
```

Comment:
```suggestion
        $this->baseUrl = $this->docConfig->get('base_url', config('app.url'));
```

Using `??` will default if the value on the left is not set, not if it's empty or null. You should use `?:` or the `$default` argument of the `get` method.

Reply:
Using the `$default` argument of the `get` method won't work because this config is set as `null`, so it will return `null`.
`??` won't return the left value if it is `null`. As you can see here:
![image](https://user-images.githubusercontent.com/12385501/62480287-ca0e9f00-b785-11e9-8965-ce4317133943.png)
Also, since we're defaulting it to null, I think it makes sense to return an empty string if someone explicitly sets it to an empty string.